src/account.ts

```diff
@@ -78,33 +78,18 @@ function parseJsonFromRawResponse (response: Uint8Array): any {
 export class Account {
     readonly connection: Connection;
     readonly accountId: string;
-    private _state: AccountState;
-
-    private _ready: Promise<void>;
-    protected get ready(): Promise<void> {
-        return this._ready || (this._ready = Promise.resolve(this.fetchState()));
-    }
 
     constructor(connection: Connection, accountId: string) {
         this.connection = connection;
         this.accountId = accountId;
     }
 
-    /**
-     * Helper function when getting the state of a NEAR account
-     * @returns Promise<void>
-     */
-    async fetchState(): Promise<void> {
-        this._state = await this.connection.provider.query(`account/${this.accountId}`, '');
-    }
```

Comment:
> BREAKING CHANGE: This removes some methods and fields from `Account` which potentially can be used somewhere (like `.fetchState()`
> 
> Change potentially can be made non-breaking if we replace these by no-ops. 

Making it non-breaking and adding a `depd` deprecation warning for `fetchState` and `ready` seems easy enough. When we publish a v1, we can drop all such `depd` warnings.

Reply:
@chadoh seems good to me, gonna do it before getting out of draft